### ISSUE TITLE
Stop training after first iteration 

### ISSUE THREAD

**Issue by nabi-rony** [NVlabs/AL-MDN#14]
$ CUDA_VISIBLE_DEVICES='0,1' python train_ssd_gmm_supervised_learning.py
C:\Users\fi42\Active_learning\AL-MDN\layers\modules\l2norm.py:20: UserWarning: nn.init.constant is now deprecated in favor of nn.init.constant_.
  init.constant(self.weight,self.gamma)
Loading base network...
Initializing weights...
train_ssd_gmm_supervised_learning.py:225: UserWarning: nn.init.xavier_uniform is now deprecated in favor of nn.init.xavier_uniform_.
  init.xavier_uniform(param)
Training SSD on: VOC0712
Using the specified args:
Namespace(basenet='vgg16_reducedfc.pth', batch_size=32, cuda=True, dataset='VOC300', dataset_root='C:\\Users\\fi42\\data/VOCdevkit/', gamma=0.1, id=1, lr=0.001, momentum=0.9, num_workers=8, resume=None, save_folder='weights/', start_iter=0, visdom=False, weight_decay=0.0
005)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Active_learning\AL-MDN\utils\augmentations.py:240: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this,
 you must specify 'dtype=object' when creating the ndarray
  mode = random.choice(self.sample_options)
C:\Users\fi42\Anaconda3\envs\py36\lib\site-packages\torch\cuda\nccl.py:24: UserWarning: PyTorch is not compiled with NCCL support
  warnings.warn('PyTorch is not compiled with NCCL support')
timer: 2174.4121 sec.
iter 0 || Loss: 29.8597 || loss: 29.8597 , loss_c: 20.1772 , loss_l: 9.6825 , lr : 0.0000
**************************************************

I am using the VOC2007 dataset to train, but it stopped without throwing any error after iteration 0. I didn't change anything in the code. It took a long time to start. What might be the issue?
